std_msgs/Header header

sensor_msgs/LaserScan scan

float32 car_w
float32 car_l

int32 grid_size
int8[] grid
float32 zoom

int32 steer_l

float32 throttle
float32 steer
float32 speed
float32 acc
geometry_msgs/Pose pose